cardano-api/src/Cardano/Api/Certificate.hs

```diff
@@ -116,6 +118,10 @@ instance HasTextEnvelope Certificate where
       GenesisKeyDelegationCertificate{}       -> "Genesis key delegation"
       MIRCertificate{}                        -> "MIR"
 
+data MIRTarget =
+     StakeAddressesMIR [(StakeCredential, Lovelace)]
+   | SendToOppositePotMIR Lovelace
```

Comment:
`SendToOppositePotMIR` could mean we are transferring from the treasury to the reserves or vice versa?

Reply:
exactly

Reply:
So how does one know which way the ADA is moving?

Reply:
`MIRCert ReservesMIR (SendToOppositePotMIR $ Coin 1)` would transfer 1 lovelace from the reserves to the treasury. and `MIRCert TreasuryMIR (SendToOppositePotMIR $ Coin 1)` would transfer 1 lovelace from the treasury to the reserves.

The existing functionality was using the `MIRPot` as the source of the lovelace, so it's consistent with that. Perhaps we could change the name to `MIRSourcePot` to make it even more clear (not in this PR, but later).

Reply:
actually, here in the cardano-api changing the names is totally easy (I just don't want to have to make another change to the cardano-ledger-specs repo right now). I'm open whatever names folks find the most clear!

Reply:
I left some docstrings, hopefully it's clearer now. Of course, I'm happy if someone else want to rename the variable and explain things differently!